src/Evolution/DiscontinuousGalerkin/NormalVectorTags.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <optional>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Tags used on the interior faces of the elements
+namespace evolution::dg::Tags::InternalFace {
+/// The magnitude of the unnormalized normal covector to the interface
+struct MagnitudeOfNormal : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/// The normal covector to the interface
+template <size_t Dim>
+struct NormalCovector : db::SimpleTag {
```

Comment:
In other places in the code, like StrhalkorperTags, we called this NormalOneForm, not NormalCovector

Reply:
I prefer covector :shrug: The repo is split